corehq/apps/users/models.py

```diff
@@ -1320,241 +1320,7 @@ def fn(domain=None, data=None):
             self.__class__.__name__, item))
 
 
-class LocationUserMixin(DocumentSchema):
```

Comment:
this mixin was used in exactly one place which seemed silly so i just merged it with the user moel